src/cohere/client.py

```diff
@@ -185,17 +197,107 @@ def embed(
     delete_connector: Never = moved_function("delete_connector", ".connectors.delete")
     oauth_authorize_connector: Never = moved_function("oauth_authorize_connector", ".connectors.o_auth_authorize")
 
+    def tokenize(
+        self,
+        *,
+        text: str,
+        model: typing.Optional[str] = OMIT,
+        request_options: typing.Optional[RequestOptions] = None,
+        offline: bool = True,
+    ) -> TokenizeResponse:
+        """
+        This function splits input text into smaller units called tokens using byte-pair encoding (BPE). To learn more about tokenization and byte pair encoding, see the tokens page.
+
+        Parameters:
+            - text: str. The string to be tokenized, the minimum text length is 1 character, and the maximum text length is 65536 characters.
+
+            - model: typing.Optional[str]. An optional parameter to provide the model name. This will ensure that the tokenization uses the tokenizer used by that model.
+
+            - request_options: typing.Optional[RequestOptions]. Request-specific configuration.
+
+            - offline: typing.Optional[bool]. An optional parameter to specify whether to use an offline tokenizer. If set to True, the tokenizer config will be downloaded (and cached),
+                and the request will be processed using the offline tokenizer. If set to False, the request will be processed using the API. The default value is True.
+        ---
+        from cohere.client import AsyncClient
 
-class AsyncClient(AsyncBaseCohere):
+        client = Client(
+            client_name="YOUR_CLIENT_NAME",
+            token="YOUR_TOKEN",
+        )
+        client.tokenize(
+            text="tokenize me! :D",
+            model="command",
+        )
+        """
```

Comment:
i think you can delete this because the docstring will get inheritted from the auto generated class

Reply:
oh it does? :o there is a problem though with "appending to the docstring" for the `offline` param

Reply:
we could try this? https://stackoverflow.com/a/2025599

Reply:
wouldn't work because we need to append halfway through the parent doc for `parameters` section 😅
I'm okay completely removing it, and not having the `offline` param documented at all (I added it as an extra thing in case someone **needs** to hit the endpoint instead.

Reply:
yeah i think drop it for now so we can lean on the autogenerated one

Reply:
done